Meta/Lagom/Fuzzers/FuzzGIFLoader.cpp

```diff
@@ -12,7 +12,9 @@
 
 extern "C" int LLVMFuzzerTestOneInput(uint8_t const* data, size_t size)
 {
+#ifndef GIF_DEBUG
     AK::set_debug_enabled(false);
+#endif
```

Comment:
As far as I know, all debug flags will always be defined by CMake, they just get either a true or false value, so this will never run. So you want to use an `if constexpr (!GIF_DEBUG)` instead I think.